install/upgrade_to_2.14.php

```diff
@@ -43,6 +43,17 @@ public function upgrade(Migration $migration) {
        $this->addPropertiesToCategories();
        $this->addTargetActorUnicity();
        $this->addTargetContract();
+       $this->addEntityOption();
+   }
+
+   public function addEntityOption() {
+      global $DB;
+      $table = 'glpi_plugin_formcreator_entityconfigs';
+
+      if (!$DB->fieldExists($table, 'home_page')) {
+         $this->migration->addField($table, 'home_page', 'integer', ['after' => 'tile_design', 'value' => '-2']);
+         $this->migration->addPostQuery("UPDATE `glpi_plugin_formcreator_entityconfigs` SET `home_page`= 1 WHERE `entities_id` = 0");
+      }
```

Comment:
Migration class provides a way to update a field after it has been added. This allows to remove the if() making code a bit more readable. See Migration::addField()

Might a good idea to unit test it in tests/1-install/Config.php (along other entity settings by the way)

```suggestion
      $this->migration->addField(
         $table, 
         'home_page', 
         'integer', [
            'after' => 'tile_design', 
            'value' => '-2',
            'update' => '1',
            'condition' => 'WHERE `entities_id` = 0'
         ]
      );
```